cmd/adapter/adapter.go

```diff
@@ -137,6 +143,8 @@ func (cmd *PrometheusAdapter) addFlags() {
 		"Optional file containing the bearer token to use when connecting with Prometheus")
 	cmd.Flags().StringArrayVar(&cmd.PrometheusHeaders, "prometheus-header", cmd.PrometheusHeaders,
 		"Optional header to set on requests to prometheus-url. Can be repeated")
+	cmd.Flags().StringVar(&cmd.PrometheusVerb, "prometheus-verb", cmd.PrometheusVerb,
+		"HTTP Verb to set on requests to Prometheus.")
```

Comment:
Could you specify the default value as well as the possible values for the verbs?